.gitignore

```diff
@@ -32,6 +32,9 @@
 !setup/createVirtualEnvPy27.sh
 !setup/paths.sh
 
+# Don't ignore channel maps
+!mapping/channel-map-*.txt
```

Comment:
Right now we are not storing this info in the repo even for the v2b electronics.  Not sure if it is relevant to keep this info here for the v3.